Procfile

```diff
@@ -1,4 +1,2 @@
-web: bundle exec puma -C config/puma.rb
+web: bundle exec thrust bin/rails server
```

Comment:
I prefer to use `bin/thrust bundle exec puma -C config/puma.rb`. 
`config/puma.rb` has already some configs. So, I don't want to change existing configs. @pglombardo what do you think?

Reply:
I agree completely although I realized recently that Rails 8 uses puma by default and auto loads the `config/puma.rb` file.  So nothing is really changed here.  I've tested and puma is still booted with the settings in `config/puma.rb` loaded when calling `rails server`

Reply:
:+1:  I haven't changed `bin/rails server` part. But, I have updated `thrust` as `bin/thrust`.